auth_server.py

```diff
@@ -78,12 +78,11 @@ def auth():
         auth_result_code = 401
         # return render_template('error_page.html', error_code=auth_result_code)
```

Comment:
Should we return here to avoid setting headers that will be "empty" after failing to make it through the try/except loop?